plugins/accessibility/index.js

```diff
@@ -17,12 +17,37 @@ var q = require('q'),
  *      }]
  *    }
  *
+ *    // Tenon.io:
+ *
+ *    //  Read about the Tenon.io settings and API requirements:
+ *    //  -http://tenon.io/documentation/overview.php
+ *    //  -http://tenon.io/documentation/understanding-request-parameters.php
+ *
+ *    exports.config = {
+ *      ...
+ *      plugins: [{
+ *        tenonIO: {
+ *          options: {
```

Comment:
I like it. I did keep `printAll` outside of that options object because it's related to Protractor's output, not Tenon. I wanted to provide a way for developers to see the entire response for debugging purposes.